src/components/MessageDisplay.svelte

```diff
@@ -15,16 +15,18 @@
     channelFilters,
     livetlFontSize,
     showTimestamp,
+    ytcDeleteBehaviour
   } from '../js/store.js';
   import {
-    TextDirection
+    TextDirection,
+    YtcDeleteBehaviour
   } from '../js/constants.js';
 
   import IntroMessage from './IntroMessage.svelte';
 
   $: document.body.style.fontSize = Math.round($livetlFontSize) + 'px';
   export let direction;
-  /** @type {{ text: String, author: String, timestamp: String }[]}*/
+  /** @type {{ text: String, author: String, timestamp: String, authorId: string, messageId: string, hidden: boolean, messageArray: any[], deleted: boolean }[]}*/
```

Comment:
You could import Message from the types.js and extend the type so you don't need to use `any[]`

Reply:
I'm importing MessageItem instead since `Message` is already used by the svelte component, and if I alias it intellisense will just make all of it `any` since its still just an invalid js import.